Makefile

```diff
@@ -261,6 +261,15 @@ else
 endif
 	./scripts/test.sh "$(GO)" "$(GOFLAGS)" "$(ALL_PACKAGES)" "$(TESTS)" "$(TESTFLAGS)" "$(GOBIN)"
 
+unit-test-server: start-docker go-junit-report do-cover-file ## Runs tests.
+ifeq ($(BUILD_ENTERPRISE_READY),true)
+	@echo Running all tests
+	$(GO) test $(GOFLAGS) $(TESTFLAGS) $(ALL_PACKAGES)
```

Comment:
I was expecting a `-short` or something here. It seems like this just runs the full tests anyway?

Reply:
nope, the `TESTFLAGS` variable is including the `-short` parameter. Probably we can remove the variable here and use directly the `-short` flag

Reply:
I think it's not obvious that `test-server` by default uses `-short` and doesn't run all tests.  Also `go-junit-report do-cover-file` are not needed for `unit-test-server`.

Reply:
Should we make it more obvious what is happening? Just have `make test` run all the tests and if you want the quick version you run this target?

Reply:
Yes, I think that should be the approach. I'll do it.